### PR TITLE
clang-tidy: fix hicpp-member-init

### PR DESCRIPTION
**Pull request by dagar**

### CODE REVIEW

File: src/ashtech.h [hunk 1]
```diff
@@ -121,7 +121,7 @@ class GPSDriverAshtech : public GPSBaseStationSupport
 	uint64_t _last_timestamp_time{0};
 
 	NMEADecodeState _decode_state{NMEADecodeState::uninit};
-	uint8_t _rx_buffer[ASHTECH_RECV_BUFFER_SIZE];
+	uint8_t _rx_buffer[ASHTECH_RECV_BUFFER_SIZE] {};
```
Comment (bkueng) [inline src/ashtech.h]: This is one of the places I'd rather keep that way - It's just wasting cycles.

File: src/ubx.cpp [hunk 2]
```diff
@@ -1394,7 +1394,7 @@ GPSDriverUBX::payloadRxDone()
 		    && (_buf.payload_rx_nav_pvt.valid & UBX_RX_NAV_PVT_VALID_VALIDTIME)
 		    && (_buf.payload_rx_nav_pvt.valid & UBX_RX_NAV_PVT_VALID_FULLYRESOLVED)) {
 			/* convert to unix timestamp */
-			struct tm timeinfo;
+			struct tm timeinfo {};
 			memset(&timeinfo, 0, sizeof(timeinfo));
```
Comment (bkueng) [inline src/ubx.cpp]: Please remove this.

File: src/ashtech.h (hunk 1 above)
Comment (dagar) [reply src/ashtech.h]: Yes that's fine, this is just an experiment to see if we could have this enforced everywhere with a tool at the cost of occasional wasted effort. The current orb patterns are the main thing in the way that would actually be cost prohibitive.

**Comment by DonLakeFlyer** [comment]
Anything here need to come to QGC?

**Comment by bkueng** [comment]
No, this does not fix anything.

**Pull request merged**